### DISCUSSION TITLE
Welcome to TIDAL

### DISCUSSION THREAD

**Discussion by stefanmoro** [tidal-music/discussions#1] [Announcements]
👋  Welcome to TIDAL community discussions!

TIDALs journey towards a more open approach to development is just getting started, and we’re committed to continue building out what we make publicly available to the developer community.

Take some time to explore what we’ve made available so far, both here on [GitHub](https://github.com/orgs/tidal-music) and on [TIDAL Developer](https://developer.tidal.com). Then take to this forum to tell us what you’d like to see next. TIDAL community discussions is the place to ask questions, come with suggestions, report bugs or initiate any other form of technical discussion relating to the TIDAL Developer Platform. Just create a new discussion in a suiting category to get started.

How to get the most out of TIDAL community discussions:
* For questions relating to code in a specific repository, please open an issue directly in that repository.
* For questions not relating to TIDAL Developer Platform, such as questions regarding our existing products, your TIDAL account, subscriptions etc, please visit our main TIDAL support space at https://support.tidal.com/.

**Comment by dk2600dk-netizen** [comment]
I'm new and don't know anything about computing. However I have used this link for years https://xythium.github.io/tidal.html
The link is now complete dead, It's extremly useful,  I can fill the artist name and get all his releases by name often with 3 different entries that when opening will give me the complete content of the release being a single or album often 3 version for soundquality high, max, dolby atmos.
I get all the informations of the entry artist, song titel, composer,producer, musicians, timing and the isrc number and releasenumber.,

So when using I will have all the artists, the songs, titles and isrc numbers and finally the releasenumber of the release. But keep in mind that the releasenumber is for a digital releases and thus not found when trying to google or go to discogs.com or others o9r order it from your shop.

I believe that somebody here also use this link and thus can help getting it going again

So please help me abnd many others I presume, since there's no help to get on the above ling

**Comment by andy-js** [comment]
@stefanmoro Is the TIDAL Developer Platform just for personal use, or is there a path that can be followed to ship apps for broader public consumption/commercial use?  I'm asking because there are numerous requests from developers asking to have their access upgraded and it didn't seem like any of them got a response.
